src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManager.cs

```diff
@@ -261,7 +267,9 @@ private void LogMessage(TestMessageLevel testMessageLevel, string message)
         private void InitializeExtensions(IEnumerable<string> sources)
         {
             var extensions = TestPluginCache.Instance.GetExtensionPaths(TestPlatformConstants.TestAdapterEndsWithPattern, this.skipDefaultAdapters);
-            extensions = extensions.Concat(TestPluginCache.Instance.GetExtensionPaths(TestPlatformConstants.DataCollectorEndsWithPattern, true)).ToList();
+            
+            // remove this line once we figure out why coverlet inrpoc DC is not initialized via runsetting inproc node.
+            extensions = extensions.Concat(TestPluginCache.Instance.GetExtensionPaths(ProxyExecutionManager.CoverletDataCollector, true)).ToList();
```

Comment:
Assuming the LUT inproc data collector gets initialized via runsettings

Reply:
this code was recently added, so LUT should not be dependent on it, will double check though

Reply:
LUT inproc datacollector would be providing full codebase path in runsettings, which wouldn't depend on this flow.